README.md

```diff
@@ -21,8 +21,9 @@ Navigate to `src/models/setup.js`, edit the database name, username or password.
 
 You can also add these credentials to the `.env` file.
 ```javascript
-  export DATABASE_URL = "postgres://user:pass@example.com:5432/dbname"
-  export TEST_DB = "postgres://user:pass@example.com:5432/testdb"
+  DATABASE_URL="postgres://user:pass@example.com:5432/dbname"
+  TEST_DB="postgres://user:pass@example.com:5432/testdb"
+  NODE_ENV="development"
```

Comment:
Also, this might also be initiated in the `NODE_ENV="development" npm start` script. 
Though, leave it here as it is somehow useful if a user removes the one in the scripts that run.